scripts/compose.js

```diff
@@ -0,0 +1,147 @@
+    d.getFullYear(),
+    ('0' + (d.getMonth() + 1)).slice(-2),
+    ('0' + d.getDate()).slice(-2),
+  ].join('-')
+  const tagArray = answers.tags.split(',')
+  tagArray.forEach((tag, index) => (tagArray[index] = tag.trim()))
+  const tags = "'" + tagArray.join("','") + "'"
+
+  let frontMatter = dedent`---
+  title: ${answers.title ? answers.title : 'Untitled'}
+  date: '${date}'
+  type: ${answers.type}
+  tags: [${answers.tags ? tags : ''}]
+  cover: /img/posts/may-2021-at-asyncapi/cover.webp
+  draft: ${answers.draft === 'yes' ? true : false}
+  authors:
+    - name: Lukasz Gornicki
+      photo: /img/avatars/lpgornicki.webp
+      link: https://twitter.com/derberq
+      byline: AsyncAPI Maintainer and Community Guardian
+  excerpt: ${answers.excerpt ? answers.excerpt : ' '}
+  ---
+
+  Write your blog post content here, just remember to mention "AsyncAPI" :smile: 
+
+  **Authors**
+
+  Before submitting your blog post, don't forget to change the \`authors\` array field to include yourself :smile:
+
+  **Images**
+ 
+  Please make sure to give credit to source of the image, for example:
+  
+  > Cover image by <a href="https://pixabay.com/users/silviarita-3142410/?utm_source=link-attribution&amp;utm_medium=referral&amp;utm_campaign=image&amp;utm_content=2634391">silviarita</a> from <a href="https://pixabay.com/?utm_source=link-attribution&amp;utm_medium=referral&amp;utm_campaign=image&amp;utm_content=2634391">Pixabay</a>
+  
+  Add a cover image, you can change it on the metadata field above called \`cover\`. In case you need some inspiration we recommend https://unsplash.com/.
+  All images should be stored in the \`public/img/posts/\` folder, below is an example of using an image in your post:
+
+  ![Figure 3: APIs give developer teams more independence](/img/posts/openapi-vs-asyncapi-burning-questions/asyncapi-openapi-post_pic-03.webp)
+
+  \`![Figure 3: APIs give developer teams more independence](/img/posts/openapi-vs-asyncapi-burning-questions/asyncapi-openapi-post_pic-03.webp)\`
+
+  Also, make sure to have the following:
+    * **Compress the image as much as possible**, we recommend https://squoosh.app/
+    * The output format needs to be `.webp`
+    * Include a clear `alt` description for people that cannot see images
+
+  **Twitter**
+  
+  To embed a tweet on your post you can use a `TwitterTweetEmbed` React component, like so:
+  
+  <TwitterTweetEmbed
+    tweetId='1384127726861258756'
+    options={{
+      cards: 'hidden',
+      width: 500,
+      align: 'center'
+    }}
+  />
+  
+  **YouTube**
+  
+  To embed a YouTube video use the `YouTube` React component, like so:
+  
+  <YouTube id="yILtksZriqA" /> 
+
+  `
+
+  frontMatter = frontMatter + '\n---'
+
+  return frontMatter
+}
+
+inquirer
+  .prompt([
+    {
+      name: 'title',
+      message: 'Enter post title:',
+      type: 'input',
+    },
+    {
+      name: 'extension',
+      message: 'Choose post extension:',
+      type: 'list',
+      choices: ['md'],
```

Comment:
`Choose post extension`, is there other extensions they can use? If it's just one (which is `md`) shall we remove this question and assume it's always `md`? Unless I'm missing something?